lib/manager.js

```diff
@@ -123,6 +123,11 @@ Manager.prototype.execute = function execute (message) {
 };
 
 
+Manager.prototype.getErrorHandler = function getErrorHandler (policy) {
+  return errorPolicyMap[policy];
```

Comment:
I want consumers of this function to be clear that they provided something wrong. Then it is on them to take some other action. For example, the `Manager` constructor calls this function and throws if there is no match. While the Policy plugin falls back to using `manager.error()`.